Latest master breaks the `mandelbrot` example in c++20 with use of deleted template method

With current master 80fadd8 I get a new error in the mandelbrot example. This is with clang 14.0.1 on Ubuntu 20.04 x86_64. I did not test other compilers, let me know if that would help.

Build error message:
```
FAILED: examples/CMakeFiles/mandelbrot.dir/mandelbrot.cpp.o 
/data/user/Tools/lib/ccache/clang++  -I/home/user/xsimd/include -DDEBUG  -march=skylake -std=c++20 -glldb -fno-omit-frame-pointer -O0 -ftime-trace -O3 -march=native -mtune=native -std=c++14 -g -fPIE -std=c++20 -MD -MT examples/CMakeFiles/mandelbrot.dir/mandelbrot.cpp.o -MF examples/CMakeFiles/mandelbrot.dir/mandelbrot.cpp.o.d -o examples/CMakeFiles/mandelbrot.dir/mandelbrot.cpp.o -c /home/user/xsimd/examples/mandelbrot.cpp
/home/user/xsimd/examples/mandelbrot.cpp:280:9: warning: expression result unused [-Wunused-value]
        expand_type { (run_arch<Arch>(bencher, x0, y0, x1, x1, width, height, maxIters, buffer), 0)... };
        ^           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/user/xsimd/examples/mandelbrot.cpp:330:51: note: in instantiation of function template specialization 'run_archlist<xsimd::arch_list<xsimd::fma3<xsimd::avx2>, xsimd::avx2, xsimd::fma3<xsimd::avx>, xsimd::avx, xsimd::sse4_2, xsimd::sse4_1, xsimd::ssse3, xsimd::sse3, xsimd::sse2>>::run<pico_bench::Benchmarker<std::chrono::duration<long, std::ratio<1, 1000>>>>' requested here
    run_archlist<xsimd::supported_architectures>::run(bencher, x0, y0, x1, y1, width, height, maxIters, buf);
                                                  ^
/home/user/xsimd/examples/mandelbrot.cpp:129:42: error: attempt to use a deleted function
                int_batch_type prev_data(output + base_index);
                                         ^
/home/user/xsimd/examples/mandelbrot.cpp:249:35: note: in instantiation of function template specialization 'xsimd::mandelbrot<xsimd::fma3<xsimd::avx2>>' requested here
                         { xsimd::mandelbrot<arch>(x0, y0, x1, y1, width, height, maxIters, buffer.data()); });
                                  ^
/home/user/xsimd/examples/mandelbrot.cpp:280:24: note: in instantiation of function template specialization 'run_arch<xsimd::fma3<xsimd::avx2>, pico_bench::Benchmarker<std::chrono::duration<long, std::ratio<1, 1000>>>>' requested here
        expand_type { (run_arch<Arch>(bencher, x0, y0, x1, x1, width, height, maxIters, buffer), 0)... };
                       ^
/home/user/xsimd/examples/mandelbrot.cpp:330:51: note: in instantiation of function template specialization 'run_archlist<xsimd::arch_list<xsimd::fma3<xsimd::avx2>, xsimd::avx2, xsimd::fma3<xsimd::avx>, xsimd::avx, xsimd::sse4_2, xsimd::sse4_1, xsimd::ssse3, xsimd::sse3, xsimd::sse2>>::run<pico_bench::Benchmarker<std::chrono::duration<long, std::ratio<1, 1000>>>>' requested here
    run_archlist<xsimd::supported_architectures>::run(bencher, x0, y0, x1, y1, width, height, maxIters, buf);
                                                  ^
/home/user/xsimd/include/xsimd/config/../types/../types/xsimd_batch.hpp:221:9: note: 'batch_bool<int>' has been explicitly marked deleted here
        batch_bool(Tp const*) = delete;
        ^
```

--- a/.github/workflows/benchmark.yml
+++ b/.github/workflows/benchmark.yml
@@ -1,4 +1,4 @@
-name: benchmark
+name: benchmark & examples
 on: [push, pull_request]
 jobs:
   build:
@@ -11,7 +11,7 @@ jobs:
     - name: Setup
       run: |
         mkdir _build
-        cd _build && cmake .. -DBUILD_BENCHMARK=ON -DCMAKE_BUILD_TYPE=Release
+        cd _build && cmake .. -DBUILD_BENCHMARK=ON -DBUILD_EXAMPLE=ON -DCMAKE_BUILD_TYPE=Release
     - name: Build
       run: cmake --build _build
     - name: Testing sequential

--- a/examples/mandelbrot.cpp
+++ b/examples/mandelbrot.cpp
@@ -126,7 +126,7 @@ namespace xsimd
 
                 // implement masked store!
                 // xsimd::store_aligned(result, output + base_index, active);
-                int_batch_type prev_data(output + base_index);
+                int_batch_type prev_data = int_batch_type::load_unaligned(output + base_index);
                 select(bool_cast(active), result, prev_data)
                     .store_aligned(output + base_index);
             }

